### PR TITLE
Fix v1.4.0 bugs

### LINKED ISSUE #213: Update email on About Us page
Screenshot 2020-11-06 at 10 22 50 AM

### LINKED ISSUE #219: Shift list does not update role requirements after using shift-edit INDEX r/NEW_ROLE NEW_QUANTITY if the shift-edit removes an assignment
Shift list does not update role requirements after using shift-edit INDEX r/NEW_ROLE NEW_QUANTITY

steps to reproduce
1. assign worker to shift for a particular role
2. shift-edit INDEX r/NEW_ROLE NEW_QUANTITY to remove that particular role
3. assignment is removed but the shift list is not updated with new role

further inspection suggests that the shift still retains the original role requirement

NOT UPDATING SHIFT LIST

### LINKED ISSUE #222: Shift list does not update role requirements after role-delete if the role-delete removes an assignment
Probably linked to #219 
before role-delete
image
after role-delete:
image

### LINKED ISSUE #229: Update in UG that if a prefix that should only be used once is used multiple times, only the last occurrence is taken
Update in UG for some commands that take in prefixes that should only be used once, if these prefixes are used multiple times, only the last occurrence is taken.

### LINKED ISSUE #235: Valid role parsed as invalid with double space between index and role
Screenshot 2020-11-06 at 7 10 09 PM

### PR DESCRIPTION
**Pull request by wakululuu**
- [X] Update About Us (resolves #213)
- [X] Fix bug where role requirements are not updated (resolves #219, resolves #222)
- [X] Update UG (resolves #229)
- [X] Fix incorrect parsing of double spaces in role name (resolves #235)

### CODE REVIEW

File: src/main/java/mcscheduler/logic/commands/CancelLeaveCommand.java [hunk 1]
```diff
@@ -31,7 +31,7 @@
             + "specified shift by the index numbers used in the last worker and shift listings. "
             + "\nParameters: "
             + PREFIX_SHIFT + "SHIFT_INDEX (must be a positive integer) "
-            + "[" + PREFIX_WORKER + "WORKER_INDEX (must be a positive integer)]...\n"
+            + PREFIX_WORKER + "WORKER_INDEX (must be a positive integer)...\n"
```
Comment (sigmund-c) [inline src/main/java/mcscheduler/logic/commands/CancelLeaveCommand.java:34]: `cancel leave` can take multiple workers at once, which should be signified by square brackets?
```suggestion
            + "[" + PREFIX_WORKER + "WORKER_INDEX (must be a positive integer)]...\n"
```

File: src/main/java/mcscheduler/logic/commands/TakeLeaveCommand.java [hunk 2]
```diff
@@ -31,7 +31,7 @@
             + "during the specified shift by the index numbers used in the last worker and shift listings. "
             + "\nParameters: "
             + PREFIX_SHIFT + "SHIFT_INDEX (must be a positive integer) "
-            + "[" + PREFIX_WORKER + "WORKER_INDEX (must be a positive integer)]...\n"
+            + PREFIX_WORKER + "WORKER_INDEX (must be a positive integer)...\n"
```
Comment (sigmund-c) [inline src/main/java/mcscheduler/logic/commands/TakeLeaveCommand.java:34]: same as `cancel-leave`, should still have square brackets as can take multiple workers?
```suggestion
            + "[" + PREFIX_WORKER + "WORKER_INDEX (must be a positive integer)]...\n"
```

**Review by sigmund-c — CHANGES_REQUESTED** [review]

File: src/main/java/mcscheduler/logic/commands/CancelLeaveCommand.java (hunk 1 above)
Comment (wakululuu) [reply src/main/java/mcscheduler/logic/commands/CancelLeaveCommand.java:34]: From my understanding, square brackets signify a param is optional. Taking multiple workers at once is accounted for by `...`.

File: src/main/java/mcscheduler/logic/commands/TakeLeaveCommand.java (hunk 2 above)
Comment (wakululuu) [reply src/main/java/mcscheduler/logic/commands/TakeLeaveCommand.java:34]: Same as my reply above as well

File: src/main/java/mcscheduler/logic/commands/CancelLeaveCommand.java (hunk 1 above)
Comment (sigmund-c) [reply src/main/java/mcscheduler/logic/commands/CancelLeaveCommand.java:34]: ahh okay, then i think just need to change for UnassignCommand's usage message as it still uses square brackets, then can already

**Review by sigmund-c — APPROVED** [review]

**Pull request merged**